src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js

```diff
@@ -59,7 +59,7 @@ function CalCalendarImportController(
   function submit() {
     var notificationMessages = {
       progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
+      success: 'An email will be sent when the import is finishing',
```

Comment:
```suggestion
      success: 'You will receive an email when the import is finished',
```

Reply:
Based on other ux example , will be better to you this format ' an email will .... '

Reply:
Yes, an email will be sent to who? It is not specified in the string.